src/app.js

```diff
@@ -1 +1,42 @@
 'use strict';
+
+const minimist = require('minimist');
+
+const inputArray = minimist(process.argv.slice(2))._;
```

Comment:
i'd de-structure operation and args
```suggestion
const [operation, ...args] = minimist(process.argv.slice(2))._;
```